pandas/tseries/tests/test_tslib.py

```diff
@@ -722,6 +722,8 @@ def test_parsing_timezone_offsets(self):
             '2013-01-01T08:00:00.000000000+0800',
             '2012-12-31T16:00:00.000000000-0800',
             '12-31-2012 23:00:00-01:00',
+            '2013-01-01 05:45+05:45', #GH11708
+            '2013-01-01 05:30+05:30', #GH11708
```

Comment:
this is fine, but we also need a test asserting that these are the correct times when localized (e.g. like the example in the original bug report)